android/java/org/chromium/chrome/browser/BraveRewardsHelper.java

```diff
@@ -654,6 +652,6 @@ public static void setSpan(
     }
 
     public static boolean shouldShowNewRewardsUI() {
-        return ChromeFeatureList.isEnabled(BraveFeatureList.NEW_REWARDS_UI_FEATURE);
+        return true;
```

Comment:
Unused Java front-end code will be removed in https://github.com/brave/brave-browser/issues/45591.